### PR TITLE
[ci] Add generation of binary package artifacts.

### PR DESCRIPTION
**Pull request by Axel-Naumann**
# This Pull request:

## Checklist:

- [ ] tested changes locally
- [ ] updated the docs (if necessary)

This PR fixes #

### CODE REVIEW

**Comment by Axel-Naumann** [comment]
Failures seem unrelated, and much of this will only be "tested" once it's merged...

File: .github/workflows/root-ci-config/build_root.py [hunk 1]
```diff
@@ -348,6 +363,17 @@ def build(options, buildtype):
 
     cmake_build(buildtype)
 
+
+@github_log_group("Create binary packages")
+def create_binaries(buildtype):
```
Comment (dpiparo) [inline .github/workflows/root-ci-config/build_root.py:368]: Why is this function needed only now?

File: .github/workflows/root-ci.yml [hunk 2]
```diff
@@ -30,6 +30,10 @@ on:
         required: true
         default: master
       incremental:
+        type: boolean
+        required: true
+        default: false
```
Comment (dpiparo) [inline .github/workflows/root-ci.yml:35]: Why are we changing the default to false?

File: README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 ## About
 
-The ROOT system provides a set of OO frameworks with all the functionality
+The ROOT system provides a set of modules with all the functionality
```
Comment (dpiparo) [inline README.md:5]: Good change.

**Review by dpiparo — CHANGES_REQUESTED** [review]
Thanks for this PR. I am not requiring changes but before approving I would like to better understand a few things, about which I asked questions in the review.

File: .github/workflows/root-ci-config/build_root.py (hunk 1 above)
Comment (Axel-Naumann) [reply .github/workflows/root-ci-config/build_root.py:368]: The new CI never had the ability to create binary packages. This PR introduces this feature.

**Review by dpiparo — APPROVED** [review]
lgtm

**Pull request merged**